charts/consul/templates/client-daemonset.yaml

```diff
@@ -387,6 +387,7 @@ spec:
                 -recursor={{ quote $value }} \
                 {{- end }}
                 -config-file=/consul/extra-config/extra-from-values.json \
+                -config-file=/consul/extra-config/locality.json \
```

Comment:
How does the config file get configured here? we need the init container here too right? or am i missing something?